.contributing/the-process.md

```diff
@@ -1,120 +1,128 @@
-0 - Introduction 
-=================
+# 0 - Introduction 
 
 Note that the sections are numbered but the items within each section are dot points rather than a numbered list. The two sections should be done in order, but within each section it's probably a good idea to jump around and expect to do multiple passes.
 
-1 - Planning in consultation with stakeholders 
-===============================================
+# 1 - Planning in consultation with stakeholders 
 
--   Replace *Section* below with the section name, eg. Auth
+- Replace _Section_ below with the section name, eg. Auth
 
--   Use the next available Section number (denoted x. below)
+- Use the next available Section number (denoted x. below)
 
--   Create in Drive - a folder called "x. *Section*: Sections Planning" in "Sections Planning (CL)".  In the folder: 
+- Create in Drive - a folder called "x. _Section_: Sections Planning" in "Sections Planning (CL)".  In the folder:
```

Comment:
Since you're already editing could you change "x. *Section*: Sections Planning" to "x. *Section*: Sections Planning (CL)"

Reply:
yup!!

Reply:
👍

Reply:
trev ive made this change in the new pr #295